extensions/wikia/WikiFactory/Dumps/README.md

```diff
@@ -11,15 +11,24 @@ When a user clicks "Request a dump":
 
 ### Nagios check
 
-We have a Nagios check that keeps track of a length of the queue of unprocessed dump requests. It issues the following query:
+We have a Nagios checks that keeps track of a length of the queue of pending and failed dump requests. They issue the following queries:
```

Comment:
Minor: `We have a Nagios checks that keeps track...` -> `We have Nagios checks that keep track...`